web/src/App.vue

```diff
@@ -102,3 +112,9 @@ export default class App extends Vue {
   }
 }
 </script>
+
+<style>
+.v-messages {
+  display: none;
```

Comment:
Which messages are we hiding?

Reply:
The slider component provides extra space for some kind of messages to be added. In our case, however, it causes a bit of a style mess.